app/screens/image_preview/downloader.ios.js

```diff
@@ -55,7 +69,21 @@ export default class Downloader extends PureComponent {
         }
     }
 
+    orientationDidChange = () => {
+        const {height: deviceHeight} = Dimensions.get('window');
+        const top = this.props.show ? (deviceHeight / 2) - 100 : deviceHeight;
+
+        setTimeout(() => {
```

Comment:
Is there a way to use something like Animated.sequence to do this after a short delay?